.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: required 
+dist: trusty 
+language: generic 
+compiler:
+  - gcc
+notifications:
+  email:
+    on_success: always
+    on_failure: always
+#    recipients:
```

Comment:
I personally like recipients feature because Travis will send you an email when build is done, and it's nice when you update PR. But it's up to maintainers.